### DISCUSSION TITLE
error after change version 1.23.8 to 1.24.0

### DISCUSSION THREAD

**Discussion by ikholmatov** [go-gorm/gorm#5768] [Q&A]
![image](https://user-images.githubusercontent.com/101399756/195836743-aa7a3a8a-a5c0-4eea-a53c-21f205e5563c.png)

I can't run after update version gorm.io/gorm v1.23.8 to gorm.io/gorm v1.24.0

**Comment by jinzhu** [comment]
Upgrade all packages to latest version.
